CalibTracker/SiPixelConnectivity/src/PixelToLNKAssociateFromAscii.cc

```diff
@@ -171,23 +187,37 @@ void PixelToLNKAssociateFromAscii::addConnections(
      // z-module
      int zmodule = atoi( module.substr(3,pos-3).c_str());
 
+     // until here 
+
      // place modules in connections
+     PixelBarrelName * name = new PixelBarrelName(part, layer, zmodule, ladder, phase1_);
+     PixelBarrelName * name0 = new PixelBarrelName(module0,phase1_);
+
+     if(name->name() != module0) cout<<" wrong name "<<fedId<<" "<<linkId<<" "
```

Comment:
@dkotlins sorry this escaped my attention. `cout` is not allowed anymore. Ploease use the `messageLogger` facility